main.go

```diff
@@ -29,21 +29,36 @@ func main() {
 					},
 					&cli.BoolFlag{
 						Name:  "accelerated-dht",
-						Usage: "run the accelerated dht client",
+						Usage: "run the accelerated DHT client",
 						Value: true,
 					},
+					&cli.StringSliceFlag{
+						Name:  "content-endpoints",
```

Comment:
nit: match endpoint name from /routing/v1 like we do for other two
```suggestion
						Name:  "provider-endpoints",
```